websites/api-docs-smoke-test/src/@commercetools-docs/gatsby-theme-docs/overrides/site-icon.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import SiteIcon from '@commercetools-docs/gatsby-theme-docs/src/icons/generated/HttpApiSmall';
```

Comment:
icons folder is exporting the icons, no need to look into generated:
```suggestion
import HttpApiSmall from '@commercetools-docs/gatsby-theme-docs/src/icons/HttpApiSmall';
```
But the pattern is still bad if the icons are intended for consumption from other packages.  In that case they must be exported from the root of the gatsby-theme-docs package so that you can do something like this here:
```suggestion
import HttpApiSmall from '@commercetools-docs/gatsby-theme-docs';
// or
import Icons from '@commercetools-docs/gatsby-theme-docs';

...
<Icons.HttpApiSmall>
```